src/frontend/templates/run_stats_head.html

```diff
@@ -123,7 +123,7 @@
       var maxDataRowsOr10k = Math.max(maxDataRows,10000);
       var buffer = 0.2 * (maxDataRows - minDataRows);
       buffer += 10000; //10000 data rows
```

Comment:
Maybe we can extract a variable with "10000" as we use that magic value everywhere

Reply:
Done, thanks.